proxied.d.ts

```diff
@@ -984,6 +985,12 @@ export interface Typegram<F> {
       chat_id: number | string;
     }): true;
 
+    /** Use this method to get information about custom emoji stickers by their identifiers. Returns an Array of Sticker objects. */
+    getCustomEmojiStickers(args: {
+      /** List of custom emoji identifiers. At most 200 custom emoji identifiers can be specified. */
+      custom_emoji_ids: string[];
+    }): Sticker[];
+
```

Comment:
This is not inserted at the correct place. All methods are ordered in the same way as on https://core.telegram.org/bots/api.